gazelle/logging.go

```diff
@@ -0,0 +1,13 @@
+package gazelle
+
+import "fmt"
+
+const (
+	DEBUG_ENABLED = false
+)
+
+func DEBUG(str string, args ...interface{}) {
```

Comment:
isn't there something a bit more go-idiomatic in the stdlib that could do this logging?

Reply:
Yeah I agree. I'll look into it a bit before merging.

Reply:
The first `DEBUG` I spotted above brought me here immediately. You can use either https://github.com/golang/glog or https://github.com/sirupsen/logrus.

Reply:
I'm just deleting this for now, it was basically just debug junk